connector_ecommerce/data/ecommerce_data.xml

```diff
@@ -30,7 +30,7 @@
             <field name="standard_price">0.0</field>
             <field name="type">service</field>
             <field name="name">Gift Certificate</field>
```

Comment:
Ah! I thought you wanted to remove `Gift`.
But I disagree, it is in use in English to capitalize each word. Have a look at the Odoo's demo data: "Gold Membership", "IPad Mini", "Air Flight", ...